src/NuGet.Clients/NuGet.CommandLine/NuGet.CommandLine.csproj

```diff
@@ -151,7 +151,6 @@
   </Target>
 
   <Import Project="$(BuildCommonDirectory)common.targets" />
-  <Import Project="$(BuildCommonDirectory)embedinterop.targets" />
```

Comment:
This one gives me a pause. 

NuGet.Commandline is VS only, I'm curious why it was added in the first place and whether we might be losing something by incrementing it.

Reply:
Do you mean removing, rather than incrementing?  Otherwise It appears you're commenting on something other than what I see here.

Also, NuGet.CommandLine (nuget.exe) is not used in VS/devenv.exe. Did you mean that embedinterop.targets is supposed to be for our VS projects only?  If so, I agree, I can't think of why NuGet.CommandLine would be referencing any interop assemblies, or using COM objects.

Reply:
by removing it.

I think there used to be some things that were getting embedded.

Reply:
NuGet.CommandLine references Microsoft.VisualStudio.Setup.Configuration.Interop. Looks like it needs this for ISetupInstance. I added EmbedInteropTypes for that assembly in NuGet.CommandLine.csproj. Does that seem reasonable?

Reply:
Yep! That's great!